packages/components/text-link/src/types.ts

```diff
@@ -0,0 +1,6 @@
+import { ComponentVariant } from '@contentful/f36-core';
+
+export type TextLinkVariant =
+  | Exclude<ComponentVariant, 'warning'>
```

Comment:
I personally would always state all variants inline without using `ComponentVariant` but I don't have a strong objection.

Reply:
Me neither 🙂 I just tried to keep consistency with `Button` https://github.com/contentful/forma-36/blob/37e988d17debe3499588e3b13975a347f212d824/packages/components/button/src/types.ts#L5-L7
We can talk about this as well

Reply:
yeah, let's talk